### ISSUE TITLE
Run Specified Tests 

### ISSUE THREAD

**Issue by aplssf** [joeferraro/MavensMate#655]
The choice to Run Specified Tests would save me a lot of time, particularly when validating changes.

![run specified tests](https://cloud.githubusercontent.com/assets/10284831/14060745/6fbb96ee-f365-11e5-9d6a-7eb6442a1190.jpg)

I'm using v0.0.10 of the MM app.

**Comment by pbattisson** [comment]
Hi Alex

Are you requesting this change for the standard Salesforce UI when performing a change set deployment/validation?

**Comment by aplssf** [comment]
No, for the MavensMate app.

**Comment by pbattisson** [comment]
You can already do this. Goto "Mavensmate" -> "Unit Testing" -> "Open Apex Unit Test Runner UI..."

<img width="661" alt="screen shot 2016-03-26 at 15 42 30" src="https://cloud.githubusercontent.com/assets/795556/14060888/8c0cbc70-f369-11e5-853c-62c62e28108b.png">

You can then select the test you want to run.

<img width="1014" alt="screen shot 2016-03-26 at 15 35 45" src="https://cloud.githubusercontent.com/assets/795556/14060889/93e38ed8-f369-11e5-9fd9-27483a133c69.png">

**Comment by aplssf** [comment]
@pbattisson yes you can run specific tests like that but not while validating / deploying projects. The option to run specified tests is an option when validating / deploying change sets (see my first screenshot) and it would be useful if the app incorporated this functionality.

**Comment by Blujay75** [comment]
I agree this would be really helpful, I'm currently using Mavensmate to write my code then having to use Workbench to import the package into Salesforce.

I've got a couple of unit tests from managed packages that always fail so I need to only run the tests for the project I'm working on.
